galpy/potential_src/Potential.py

```diff
@@ -1510,6 +1510,166 @@ def nemo_accpars(self,vo,ro):
+        """
+            
+        NAME:
+            
+            rtide
+            
+        PURPOSE:
+            
+            Calculate the tidal radius for object of mass M assuming a circular orbit 
+            at R in potential Pot given the formalism of Bertin, G., & Varri, A. L. 2008, ApJ, 689, 1005
+        INPUT:
+            
+            Pot - Potential instance or list of such instances
+            
+            R - Galactocentric radius (can be Quantity)
+            
+            z - height (can be Quantity)
+            
+            M - (default = None) Mass of object (can be Quantity)
+            
+        OUTPUT:
+            
+            Tidal Radius
+        
+        HISTORY:
+            
+            2018-02-12 - Written - Webb (UofT)
+            
+        """
+        if self.isNonAxi:
+            #Make sure potential is axisymmetric
+            print("Error - cannot calculate rtide for non-axisymmetric potential")
+            return 0.0
+        elif M==None:
+            #If units not given, assume galpy! GM units?
+            print("Error - mass parameter has not been set")
+            return 0.0
+        else:
+            #Check mass units (default: GALPY units)
+            if M_units=='mass':
+                M/=bovy_conversion.mass_in_msol(self._vo,self._ro)
+            elif M_units=='GM':
+                M/=(bovy_conversion.mass_in_msol(self._vo,self._ro)*bovy_conversion._G)
+            # Parse M if it has units
+            elif _APY_LOADED and isinstance(M,units.Quantity):
+                # Try a bunch of possible units
+                unitFound= False
+                # mass
+                try:
+                    M= M.to(units.Msun).value\
+                        /bovy_conversion.mass_in_msol(self._vo,self._ro)
+                except units.UnitConversionError: pass
+                else:
+                    unitFound= True
+                    if not M_units == 'mass':
+                        raise units.UnitConversionError('M= parameter of %s should have units of %s, but has units of mass instead' % (type(self).__name__,M_units))
+                if not unitFound:
+                    # G x mass
+                    try:
+                        M= M.to(units.pc*units.km**2/units.s**2)\
+                            .value\
+                            /bovy_conversion.mass_in_msol(self._vo,self._ro)\
+                            /bovy_conversion._G
+                    except units.UnitConversionError: pass
+                    else:
+                        unitFound= True
+                        if not M_units == 'mass':
+                            raise units.UnitConversionError('M= parameter of %s should have units of %s, but has units of G x mass instead' % (type(self).__name__,M_units))
+                                    
+            #To calculate omegac in spherical coordinates we need the first derivative of the potential with respect to r
```

Comment:
Switch back to the regular method indentation (you're doing this under the ``else`` clause here)